android/filament-android/src/main/java/com/google/android/filament/RenderableManager.java

```diff
@@ -809,6 +848,7 @@ public long getNativeObject() {
     private static native int nBuilderSkinningBones(long nativeBuilder, int boneCount, Buffer bones, int remaining);
     private static native void nBuilderSkinningBuffer(long nativeBuilder, long nativeSkinningBuffer, int boneCount, int offset);
     private static native void nBuilderMorphing(long nativeBuilder, int targetCount);
+    private static native void nBuilderMorphing(long nativeBuilder, int level, int primitiveIndex, long nativeMorphTargetBuffer, int offset, int count);
```

Comment:
please change the name of this one (to avoid overloads for the native JNI), for instance `nBuilderMorphingWithData`

Reply:
Do we have to use the different function name for this? It will be handled by JNI automatically if we add postfix.

Reply:
Yes but I don't want to have to deal with name mangling if we don't have to. I prefer to use a different name. This is all private code, so it doesn't matter too much if the jni implementation name doesn't match exactly the public api.

Reply:
Ok I understand but could you consider again? Because it's java compiler feature. You can generate jni header by `javac -h jni_header file.java`. You don't have to deal with name mangling. If you still want to use a different name, I will change a function name.

Reply:
@pixelflinger Ok I understand but could you consider again? Because it's java compiler feature. You can generate jni header by `javac -h jni_header file.java`. You don't have to deal with name mangling. If you still want to use a different name, I will change a function name.

Reply:
@daemyung No, those auto-generated bindings are absolutely horrible and make incremental work a major pain. Note that ideally we'd switch of all our JNI to manual registration from `JNI_OnLoad` to catch mistakes in the bindings at load time, but also to get rid of symbols and shrink the final .so.

Reply:
Ok I see what you want. I will change a function name.